webapp/store/views.py

```diff
@@ -156,12 +157,28 @@ def details(charm_or_bundle_name, series_or_version=None, version=None):
         entity = models.get_charm_or_bundle(reference)
 
     if entity:
+        try:
+            response = requests.get(
+                (
+                    f"https://api.snapcraft.io/v2/charms/info/"
+                    f"{charm_or_bundle_name}"
+                )
+            )
+            exists_charmhub = response.status_code == 200
```

Comment:
Maybe `exists_in_charmhub` is a better name?

Reply:
var renamed